### PR TITLE
Resolve #1540, #1542, #1545

### LINKED ISSUE #1540: Redirect page on maintenance
maintenance mode is on and a redirection to status page is needed to prevent additional inputs.
reference: https://medium.com/stashaway-engineering/deploying-a-self-contained-maintenance-page-on-aws-ad4ec8e0a736

### PR DESCRIPTION
**Pull request by timothywangdev**

### CODE REVIEW

File: src/apis.js [hunk 1]
```diff
@@ -13,6 +13,13 @@ const chainsferApi = axios.create({
   baseURL: env.REACT_APP_CHAINSFER_API_ENDPOINT,
   headers: {
     'Content-Type': 'application/json'
+  },
+  validateStatus: status => {
```
Comment (InVincible2016) [inline src/apis.js:17]: The server maintenance status checking should be done here: 
https://github.com/ventureum/Chainsfr/blob/master/src/App.js#L190

File: src/components/NavDrawerComponent.jsx [hunk 2]
```diff
@@ -240,6 +240,25 @@ const NavDrawerComponent = (props: Props) => {
                 </Typography>
               </Box>
             </Box>
+            <Typography variant='caption' color='textSecondary'>
+              <Box color='text.disabled' display='inline'>
```
Comment (InVincible2016) [inline src/components/NavDrawerComponent.jsx:244]: don't put `<div>` inside `<p>`

File: src/apis.js (hunk 1 above)
Comment (timothywangdev) [reply src/apis.js:17]: preload can only check when refresh

File: src/components/NavDrawerComponent.jsx (hunk 2 above)
Comment (timothywangdev) [reply src/components/NavDrawerComponent.jsx:244]: copied from line 222

**Pull request merged**